sapi/fpm/fpm/fpm_sockets.c

```diff
@@ -90,7 +90,10 @@ static void fpm_sockets_cleanup(int which, void *arg) /* {{{ */
 				}
 			}
 
-			p += sprintf(env_value + p + socket_set_buf, "%s%s=%s", (p && !socket_set_buf) ? "," : "", ls->key, fd);
+			int written = snprintf(env_value + p + socket_set_buf, 
+				strlen(ls->key) + strlen(fd) + 2 + (p && !socket_set_buf ? 1 : 0), 
```

Comment:
Might be better to store the length in a variable, and then subtract `p + socket_set_buf` from it.